content/hi/docs/contribute/review/reviewing-prs.md

```diff
@@ -0,0 +1,150 @@
+- कुबेरनेट्स प्रलेखन समुदाय में विभिन्न [भूमिकाओं और जिम्मेदारियों](/docs/contribute/participate/roles-and-responsibilities/) को समझें।
+
+<!-- body -->
+
+## शुरू करने से पहले
+
+समीक्षा शुरू करने से पहले:
+
+- [CNCF आचार संहिता](https://github.com/cncf/foundation/blob/main/code-of-conduct.md) पढ़ें और सुनिश्चित करें कि आप हर समय इसका पालन करते हैं।
+- विनम्र, विचारशील और मददगार बनें।
+- PR के सकारात्मक पहलुओं के साथ-साथ परिवर्तनों पर टिप्पणी करें।
+- आपकी समीक्षा कैसे प्राप्त हो सकती है, इस बारे में सहानुभूतिपूर्ण और सावधान रहें।
+- अच्छी मंशा मानें और स्पष्ट प्रश्न पूछें।
+- अनुभवी योगदानकर्ता, नए योगदानकर्ताओं के साथ जोड़ी बनाने पर विचार करें जिनके कार्य में व्यापक परिवर्तन की आवश्यकता है।
+
+## पुनरावलोकन प्रक्रिया
+
+सामान्य तौर पर, अंग्रेज़ी में सामग्री और शैली के लिए पुल अनुरोधों की समीक्षा करें। चित्र 1 
+समीक्षा प्रक्रिया के चरणों को रेखांकित करता है। प्रत्येक चरण का विवरण इस प्रकार है।
+
+<!-- See https://github.com/kubernetes/website/issues/28808 for live-editor URL to this figure -->
+<!-- You can also cut/paste the mermaid code into the live editor at https://mermaid-js.github.io/mermaid-live-editor to play around with it -->
+
+{{< mermaid >}}
+flowchart LR
+    subgraph fourth[समीक्षा शुरू करें]
+    direction TB
+    S[ ] -.-
+    M[टिप्पणियाँ लिखें] --> N[परिवर्तनों की समीक्षा करें]
+    N --> O[नए योगदानकर्ताओं को <br>टिप्पणी का चयन करना चाहिए]
+    end
+    subgraph third[PR का चयन करें]
+    direction TB
+    T[ ] -.-
+    J[विवरण<br>और टिप्पणियाँ पढ़ें]--> K[Netlify प्रीव्यू बिल्ड <br> के साथ अपने परिवर्तनों का पूर्वावलोकन करें]
+    end
+ 
+  A[खुली PR सूची की समीक्षा करें]--> B[खुले PR को <br>लेबल द्वारा फ़िल्टर करें]
+  B --> third --> fourth
+     
+
+classDef grey fill:#dddddd,stroke:#ffffff,stroke-width:px,color:#000000, font-size:15px;
+classDef white fill:#ffffff,stroke:#000,stroke-width:px,color:#000,font-weight:bold
+classDef spacewhite fill:#ffffff,stroke:#fff,stroke-width:0px,color:#000
+class A,B,J,K,M,N,O grey
+class S,T spacewhite
+class third,fourth white
+{{</ mermaid >}}
+
+चित्र 1. समीक्षा प्रक्रिया के चरण
+
+
+1. यहाँ [https://github.com/kubernetes/website/pulls](https://github.com/kubernetes/website/pulls) पर जाएं।
+   आप कुबेरनेट्स वेबसाइट और प्रलेखन में हर खुले पुल अनुरोध की एक सूची देख सकते हैं।
+
+2. निम्नलिखित में से एक या सभी लेबलों का उपयोग करके खुले PR को फ़िल्टर करें:
+
+   - `cncf-cla: yes` (अनुशंसित): जिन योगदानकर्ताओं ने CLA पर हस्ताक्षर नहीं किए हैं, उनके PR का विलय नहीं किया जा सकता है। अधिक जानकारी के लिए [CLA पर हस्ताक्षर करें](/docs/contribute/new-content/#sign-the-cla) देखें।
+   - `language/en` (अनुशंसित): केवल अंग्रेजी भाषा में PR देखने के लिए फ़िल्टर।
+   - `size/<size>`: एक निश्चित आकार के PR के लिए फ़िल्टर। यदि आप नए हैं, तो छोटे PR से शुरुआत करें।
+
+   इसके अतिरिक्त, सुनिश्चित करें कि PR को कार्य प्रगति में है के रूप में चिह्नित नहीं किया गया है। `work in progress` लेबल का उपयोग करने वाले पीआर अभी तक समीक्षा के लिए तैयार नहीं हैं।
```

Comment:
```suggestion
   इसके अतिरिक्त, सुनिश्चित करें कि PR को कार्य प्रगति में है के रूप में चिह्नित नहीं किया गया है। `work in progress` लेबल का उपयोग करने वाले PR अभी तक समीक्षा के लिए तैयार नहीं हैं।
```

Reply:
I would suggest that we do not use short forms and write it completely as pull request.